app/src/main/java/com/example/ithaca_transit_android_v2/Repository.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.ithaca_transit_android_v2
+
+import com.example.ithaca_transit_android_v2.models.Location
+
+object Repository {
+    init {
+    }
+    private var currentLocation: android.location.Location? = null
+    private var startLocation: Location? = null
+    private var destinationLocation: Location? = null
+
+    fun setCurrentLocation(loc: android.location.Location) {
+        this.currentLocation = loc
+    }
+
```

Comment:
don't need to create getter and setters for kotlin, it's automatically created: https://kotlinlang.org/docs/reference/properties.html